cmd/kubectl-gadget/version.go

```diff
@@ -36,6 +40,45 @@ var versionCmd = &cobra.Command{
 	Use:   "version",
 	Short: "Show version",
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println(version)
+
+		fmt.Println("Client version:", version)
+
+		client, err := k8sutil.NewClientsetFromConfigFlags(utils.KubernetesConfigFlags)
+		if err != nil {
+			return
```

Comment:
Is there a standard way to log errors or warnings in IG?
Just println like i did in my latest version of this commit?

Reply:
We use `fmt.Fprintf(os.Stderr` in the CLI to print errors. On the server part we use `"github.com/sirupsen/logrus"`.

Reply:
For both failure scenarios i found a function in common errors, so i used them.
Personally i would prefer to have the failure message change in a separate commit and PR

Reply:
Just one thing. If you use `RunE` instead of [`Run`](https://github.com/burak-ok/inspektor-gadget/blob/b4a2af37861e8556b9434651523806acf7061a64/cmd/kubectl-gadget/version.go#L44), the `cobra` library will take care of sending the error message to the `stderr`, and you just need to return what `WrapInErrSetupK8sClient` returns (what a pun 🤣).

Reply:
Done. Also seems way better :D